### PR TITLE
docs(batch): add batch startTime to API

### PR DESCRIPTION
**Pull request by danielkelemen**
related to CAM-14721

### CODE REVIEW

File: content/reference/rest/batch/get-query.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ GET `/batch`
     <td>sortBy</td>
     <td>
       Sort the results lexicographically by a given criterion.
-      Valid values are <code>batchId</code> and <code>tenantId</code>.
+      Valid values are <code>batchId</code>, <code>tenantId</code> and <code>startTime</code>.
```
Comment (ThorbenLindhauer) [inline content/reference/rest/batch/get-query.md:65]: This and the result property should go to `get-statistics-query.md`.

**Review by ThorbenLindhauer — CHANGES_REQUESTED** [review]
It looks like you updated the wrong endpoint description, please check the comment.

File: content/reference/rest/batch/get-query.md (hunk 1 above)
Comment (danielkelemen) [reply content/reference/rest/batch/get-query.md]: You're right, the new sortBy property is only available for the statistics query.
What did you mean by the result property? The new startTime field is returned for both batch and statistics endpoints.

Comment (ThorbenLindhauer) [reply content/reference/rest/batch/get-query.md]: Yeah, you are right. I don't recall why I wrote it like that.

**Review by ThorbenLindhauer — APPROVED** [review]
Looks good!

**Pull request merged**